code/datums/outfits/outfit_admin.dm

```diff
@@ -560,54 +560,17 @@
 	if(istype(I))
 		apply_to_card(I, H, list(access_maint_tunnels), "Spy", "syndie")
 
-
-
-/datum/outfit/admin/vox
```

Comment:
please do not remove this, it would be preferable that you simply remove the
```
else
 equip_vox_raider()
 regenerate_icons()
```
call from /equip. This loadout is specifically for armalises.